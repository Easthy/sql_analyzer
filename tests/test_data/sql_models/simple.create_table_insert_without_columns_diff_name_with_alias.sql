BEGIN TRANSACTION;

CREATE TABLE IF NOT EXISTS simple.create_table_insert_without_columns_diff_name_with_alias (
    user_id INTEGER,
    registration_date DATE ENCODE DELTA
)
    DISTSTYLE KEY
    DISTKEY(user_id)
    SORTKEY(registration_date)
;

INSERT INTO simple.create_table_insert_without_columns_diff_name_with_alias
SELECT mysql_users.user_id,
       mysql_users.reg_date AS registration_date
  FROM mwl._mysql_users
;

END TRANSACTION;