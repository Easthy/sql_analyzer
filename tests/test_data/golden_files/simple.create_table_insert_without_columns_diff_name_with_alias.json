{
  "nodes": [
    { "id": "col:simple.create_table_insert_without_columns_diff_name_with_alias.user_id" },
    { "id": "col:simple.create_table_insert_without_columns_diff_name_with_alias.registration_date" },
    { "id": "tbl:simple.create_table_insert_without_columns_diff_name_with_alias" },
    { "id": "tbl:mwl._mysql_users" }
  ],
  "links": [
    { "source": "tbl:simple.create_table_insert_without_columns_diff_name_with_alias", "target": "col:simple.create_table_insert_without_columns_diff_name_with_alias.user_id", "type": "contains_column" },
    { "source": "tbl:simple.create_table_insert_without_columns_diff_name_with_alias", "target": "col:simple.create_table_insert_without_columns_diff_name_with_alias.registration_date", "type": "contains_column" },
    { "source":"tbl:simple.create_table_insert_without_columns_diff_name_with_alias", "target":"tbl:mwl._mysql_users", "type":"table_dependency" }
  ]
}